package.json

```diff
@@ -1,9 +1,20 @@
 {
-  "name": "oran",
-  "version": "0.0.1",
-  "description": "Oran.js JavaScript by Orange Cats",
-  "main": "index.js",
+  "name": "myapp",
+  "version": "0.0.0",
+  "private": true,
   "scripts": {
+<<<<<<< HEAD
```

Comment:
It looks like you didn't fully resolve the merge conflicts before submitting the PR